src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,40 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+import './todoinfo.scss';
+
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    completed,
+    title,
+    user,
+  } = todo;
+
+  return (
+    <article
+      data-id={todo.id}
```

Comment:
You destructered the todo while doing it, why don't you have the id there too?

Reply:
I forgot it :(